front/src/common/Map/Layers/MastsLayers.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Shared layer showing the mast for icons on the map
+ */
+import { SymbolLayer } from 'react-map-gl/maplibre';
+import { isNil } from 'lodash';
+import { OmitLayer } from 'types';
```

Comment:
When you're importing a type, you can add the "type" keyword in front of the import :)
```suggestion
import type { SymbolLayer } from 'react-map-gl/maplibre';
import { isNil } from 'lodash';
import type { OmitLayer } from 'types';
```